packages/cli/lib/artifact-generator.js

```diff
@@ -102,4 +104,16 @@ module.exports = class ArtifactGenerator extends BaseGenerator {
       {globOptions: {dot: true}},
     );
   }
+
+  async updateIndexFile() {
+    await index(this.artifactInfo.outDir, {
+      prefix: `.${this.artifactInfo.type}`,
```

Comment:
It's weird to me to have to `.` as part of `prefix`. I think it's better for `.` to be inferred when setting `globPattern` in `make-index.js`